### ISSUE TITLE
Error when opening PDF file in android app.

### ISSUE THREAD

**Issue by itsadamslife** [nextcloud/android#7188]
Error when opening PDF file in android app.

************ CAUSE OF ERROR ************

java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.swiperefreshlayout.widget.SwipeRefreshLayout.setVisibility(int)' on a null object reference
	at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.populateList(FileDetailActivitiesFragment.java:418)
	at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.lambda$null$3$FileDetailActivitiesFragment(FileDetailActivitiesFragment.java:370)
	at com.owncloud.android.ui.fragment.-$$Lambda$FileDetailActivitiesFragment$ZzLUj33qKg0e1Ki8Vd5bki8I6B4.run(Unknown Source:6)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7656)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140051
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: google
Device: coral
Model: Pixel 4 XL
Id: RP1A.201005.004
Product: coral

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: 6782484

**Comment by Derkades** [comment]
Same crash when opening a text file:
```
************ CAUSE OF ERROR ************

java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.swiperefreshlayout.widget.SwipeRefreshLayout.setVisibility(int)' on a null object reference
	at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.populateList(FileDetailActivitiesFragment.java:415)
	at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.lambda$null$3$FileDetailActivitiesFragment(FileDetailActivitiesFragment.java:370)
	at com.owncloud.android.ui.fragment.-$$Lambda$FileDetailActivitiesFragment$ZzLUj33qKg0e1Ki8Vd5bki8I6B4.run(Unknown Source:6)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7656)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140051
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: google
Device: sargo
Model: Pixel 3a
Id: RP1A.201005.004
Product: sargo

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: 6782484
```

**Comment by Derkades** [comment]
Possible duplicate of #7080, different stacktrace but same exception and exception message

**Comment by AndyScherzinger** [comment]
Does this happen for a already downloaded file (green checkmark) or. For not yet downloaded files?

**Comment by Derkades** [comment]
Only for files that are not yet downloaded, synced files work fine.

**Comment by AndyScherzinger** [comment]
fixed via #7105 or #7212
